tests/view/rooms/list/app.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/view/rooms/list/app.js
+ */
+
+import ReactTestUtils from 'react-dom/test-utils';
+import { jsdom } from 'jsdom'
+
+const expect = require('chai').expect;
+
+const app = require('../../../../view/rooms/list/app.js');
+
+describe('tests/view/rooms/list/app.js', () => {
+  const testArr = ['1', '2', '3'];
+
+  it('ok, select element created with 3 options', () => {
+    const selectEl = app.createSelectEl(testArr)
+    expect(selectEl.tagName).to.equal('SELECT');
+    expect(selectEl.childElementCount).to.equal(testArr.length);
+    expect(selectEl.value).to.equal('1');
```

Comment:
is the first position the default value?

Reply:
Yeah if no default value is passed to createSelectEl function, the first element in the array is used as the default